src/SampleData.jl

```diff
@@ -0,0 +1,112 @@
+
+module SampleData
+
+using CelesteTypes
+import SDSS
+import OptimizeElbo
+import ModelInit
+
+using Base.Test
```

Comment:
SampleData probably shouldn't import Base.Test. Are the other imports necessary?